src/Core/wwwroot/js/dialog-utils.js

```diff
@@ -0,0 +1,7 @@
+export function getActiveElement() {
```

Comment:
We cannot accept JavaScript code in wwwroot files. You must move your code to a `FluentDialogProvider.razor.js` file. You will find an example in the `FluentButton` button.

Reply:
I thought that wasn’t supported outside of a blazor component?